crates/uv-resolver/src/pubgrub/report.rs

```diff
@@ -407,67 +407,59 @@ impl PubGrubReportFormatter<'_> {
         unavailable_packages: &FxHashMap<PackageName, UnavailablePackage>,
         incomplete_packages: &FxHashMap<PackageName, BTreeMap<Version, IncompletePackage>>,
     ) -> IndexSet<PubGrubHint> {
-        /// Returns `true` if pre-releases were allowed for a package.
-        fn allowed_prerelease(package: &PubGrubPackage, selector: &CandidateSelector) -> bool {
-            let PubGrubPackageInner::Package { name, .. } = &**package else {
-                return false;
-            };
-            selector.prerelease_strategy().allows(name)
-        }
```

Comment:
Not really necessary anymore, we just inline `selector.prerelease_strategy().allows(name)` because we have the name already.